source/agora/node/FullNode.d

```diff
@@ -1131,7 +1131,7 @@ public class FullNode : API
     public override TimePoint getLocalTime () @safe nothrow
     {
         this.recordReq("local_time");
```

Comment:
Trivial

```suggestion
        this.recordReq("utc_time");
```

Reply:
No, the endpoint is still `GET /local_time`. We can change it later but I tend to avoid those and just pack them in one commit / release.